testsuite/integration-arquillian/tests/other/console/src/main/resources/themes/qe/login/messages/messages_en.properties

```diff
@@ -0,0 +1,4 @@
+doAccept=Yes
```

Comment:
For custom themes it's a bit of a pandoras box. Testing the following could be a start:
- Add custom locale (doesn't need to add more than a few keys, as the rest will be inherited from English)
- Override a few messages
- Modify template (one is enough)
- Add stylesheet
- Add image
- Add custom attributes to registration page and account page
It would be much easier to maintain if we have a single test theme that does all of the above. It should be called CustomThemeTest or something.